src/ResourceManager/Resources/Commands.Resources/packages.config

```diff
@@ -2,4 +2,5 @@
 <packages>
   <package id="Microsoft.Azure.Management.ResourceManager" version="1.6.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Authorization" version="2.6.0-preview" targetFramework="net452" />
```

Comment:
@darshanhs90 since you are adding a new package to your module, you will need to do two additional things:

(1) Add the new package to the `RequiredAssemblies` field in [`AzureRM.Resources`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Resources/AzureRM.Resources.psd1#L57)

(2) [Update the `.wxi` file](https://github.com/Azure/azure-powershell/wiki/Azure-Powershell-Developer-Guide#updating-the-installer)

Reply:
@cormacpayne Regenerating the wxi file added a bunch of changes to that file and also introduced a build issue,any suggestions on how to get that fixed